src/main/java/mods/eln/Eln.java

```diff
@@ -6468,10 +6768,23 @@ private void recipeIngot() {
 
     private void recipeDust() {
         addShapelessRecipe(findItemStack("Alloy Dust", 2),
-            "dustIron",
             "dustIron",
             "dustCoal",
-            dictTungstenDust);
+            dictTungstenDust,
+			dictTungstenDust,
```

Comment:
Indentation

Reply:
Also, that should be "dustTungsten".

Reply:
It appears to be dictTungstenDust in the rest of the file, do you want me to change all instances of dictTungstenDust to dustTungsten?